### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by phosseini**
Adding SnorkelPrint with two methods for now, for 1-printing list of extracted candidates in documents, and 2-getting text of documents and their corresponding sentences.

### CODE REVIEW

**Comment by phosseini** [comment]
This is how the class can be used (two method for now:)
```
obj = SnorkelPrint("snorkel.db")
# printing candidates
obj.print_candidates(['my_rel', ['entity1', 'entity2']], "article-1", max_output=10)

# getting documents information
doc_text, doc_sents, doc_sents_start_idx = obj.get_doc_text("article-4")

# checking the indexes and list of sentences
text = ""
for i in range(len(doc_sents)):
    text = text + " " + doc_text[int(doc_sents_start_idx[i]):int(doc_sents_start_idx[i])+len(doc_sents[i])]
if text.strip() == doc_text.strip():
    print("true")
else:
    print("false")
```

File: snorkel/utils.py [hunk 1]
```diff
@@ -145,3 +146,124 @@ def tokens_to_ngrams(tokens, n_max=3, delim=' '):
     for root in range(N):
         for n in range(min(n_max, N - root)):
             yield delim.join(tokens[root:root+n+1])
+
+class SnorkelPrint:
+    def __init__(self, db_path):
+        self.conn = self.__create_connection(db_path)
+
+    @staticmethod
+    def __create_connection(db_path):
```
Comment (vincentschen) [inline snorkel/utils.py]: nit: can we single underscore for private methods? re: https://www.python.org/dev/peps/pep-0008/#id36

Comment (vincentschen) [reply snorkel/utils.py]: applies to private methods below

File: snorkel/utils.py [hunk 2]
```diff
@@ -145,3 +146,124 @@ def tokens_to_ngrams(tokens, n_max=3, delim=' '):
     for root in range(N):
         for n in range(min(n_max, N - root)):
             yield delim.join(tokens[root:root+n+1])
+
+class SnorkelPrint:
```
Comment (vincentschen) [inline snorkel/utils.py]: Can we rename this class to `CandidateViewer` or something more descriptive?

File: snorkel/utils.py (hunk 1 above)
Comment (phosseini) [reply snorkel/utils.py]: Good point, just changed the names.

File: snorkel/utils.py (hunk 2 above)
Comment (phosseini) [reply snorkel/utils.py]: Agreed, I changed the name to `CandidateViewer`. Feel free to change the name to a better or more descriptive one.

**Comment by phosseini** [comment]
> Overall, awesome contribution!
> I'm thinking this might fit better under `snorkel/contrib` under a separate file.
> 
> Adding @henryre — in case he had any additional comments! Feel free to re-request a review when you've made updates.

Do you mean in a `.py` class in `snorkel/contrib`? If yes, do you have recommendation for the class name?

**Comment by vincentschen** [comment]
> Do you mean in a `.py` class in `snorkel/contrib`? If yes, do you have recommendation for the class name?

That's correct— why don't we name it `CandidateViewer` for both the class and filename (`candidate_viewer.py`)?

**Comment by phosseini** [comment]
> > Do you mean in a `.py` class in `snorkel/contrib`? If yes, do you have recommendation for the class name?
> 
> That's correct— why don't we name it `CandidateViewer` for both the class and filename (`candidate_viewer.py`)?

It's a good name, just created the class and put it in the `contrib` folder

**Comment by vincentschen** [comment]
> > > Do you mean in a `.py` class in `snorkel/contrib`? If yes, do you have recommendation for the class name?
> > 
> > 
> > That's correct— why don't we name it `CandidateViewer` for both the class and filename (`candidate_viewer.py`)?
> 
> It's a good name, just created the class and put it in the `contrib` folder

@phosseini I don't see it in the PR— is this change pushed?

**Comment by vincentschen** [comment]
Nevermind, I see this: https://github.com/HazyResearch/snorkel/pull/1369

Can we close this PR?

**Comment by vincentschen** [comment]
Closing, merged via https://github.com/snorkel-team/snorkel/pull/1369

**Pull request closed without merging**